.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+# Install pre-commit hooks via
+# pre-commit install
+
+exclude: >
+    (?x)^(
+      \.vscode/settings\.json|
+    )$
+repos:
+
+  - repo: git://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+    - id: check-toml
+    - id: check-json
+    - id: check-yaml
+    - id: end-of-file-fixer
+    - id: trailing-whitespace
+
+  - repo: https://github.com/psf/black
+    rev: 22.1.0
+    hooks:
+    - id: black
+
+  - repo: https://github.com/executablebooks/mdformat
+    rev: 0.7.13
+    hooks:
+      - id: mdformat
```

Comment:
yeh so you need to use this with plugins 😉 e.g.

- https://github.com/executablebooks/mdformat-myst
- https://github.com/hukkin/mdformat-gfm (for tasklists)

Reply:
ahh ok, I will try to give that a shot

Reply:
ahhhh, ok I will take a look and figure it out

Reply:
Meaning to circle back to mdformat-myst at some point and do some more testing/improvements.
Would be nice to advertise it on jupyter-book (but only when I'm certain it is properly working)